GitHubTeams.ps1

```diff
@@ -276,6 +314,407 @@ filter Get-GitHubTeamMember
     return (Invoke-GHRestMethodMultipleResult @params | Add-GitHubUserAdditionalProperties)
 }
 
+function New-GitHubTeam
+{
+<#
+    .SYNOPSIS
+        Creates a team within an organization on GitHub.
+
+    .DESCRIPTION
+        Creates a team or within an organization on GitHub.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OrganizationName
+        The name of the organization to create the team in.
+
+    .PARAMETER TeamName
+        The name of the team.
+
+    .PARAMETER Description
+        The description for the team.
+
+    .PARAMETER Maintainers
+        A list of GitHub IDs for organization members who will become team maintainers.
+
+    .PARAMETER RepositoryName
+        A list of repositories to add the team to.
+
+    .PARAMETER Privacy
+        The level of privacy this team should have.
+
+    .PARAMETER ParentTeamName
+        The name of a team to set as the parent team.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .OUTPUTS
+        PSCustomObject
+
+    .EXAMPLE
+        New-GitHubTeam -OrganizationName PowerShell -TeamName 'Developers'
+
+        Creates a new GitHub team called 'Developers' in the 'PowerShell' organization.
+#>
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSShouldProcess', '',
+        Justification = 'Methods called within here make use of PSShouldProcess, and the switch
+        is passed on to them inherently.')]
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSReviewUnusedParameter', '',
+        Justification = 'One or more parameters (like NoStatus) are only referenced by helper
+        methods which get access to it from the stack via Get-Variable -Scope 1.')]
+    [CmdletBinding(
+        SupportsShouldProcess,
+        PositionalBinding = $false
+    )]
+    param
+    (
+        [Parameter(Mandatory, Position = 1)]
```

Comment:
Here and elsewhere, since the pipeline change, the module is now consistent with having the `Parameter` attribute separate its properties on individual lines if there are more than one.